datapackage_pipelines_assembler/nodes/basic_nodes.py

```diff
@@ -56,3 +56,42 @@ def get_artifacts(self):
             False
         )
         yield output
+
+
+class DerivedPreviewProcessingNode(BaseProcessingNode):
+    def __init__(self, available_artifacts, outputs):
+        super(DerivedPreviewProcessingNode, self).__init__(available_artifacts, outputs)
+        self.fmt = 'json'
+
+    def get_artifacts(self):
+        for artifact in self.available_artifacts:
+            if artifact.datahub_type == 'derived/csv':
+                datahub_type = 'derifdfdved/preview'
+                resource_name = artifact.resource_name + '_preview'
```

Comment:
so is it derived csv resource name + '_preview'?

Reply:
yes, so if resource name is `vix-daily` preview name would be `vix-daily_csv_priview`  (as vix-daily_csv is the name of the resource it is iterating over)